students/studentsModel.js

```diff
@@ -0,0 +1,25 @@
+module.exports = { addStudents, getStudents, removeStudent, removeAll };
+
+let students = [];
+
+function addStudents(s) {
+  if (Array.isArray(s)) {
+    students = [...students, ...s];
+    return s;
+  } else {
+    students.push(s);
+    return students.find(student => student.name === s.name);
```

Comment:
I think this would do better just returning `s`:
```javascript
  student.push(s);
  return s;
```
We know `s` was just pushed onto the array, so there's not much point in looking for it right after!